src/config.ts

```diff
@@ -71,7 +71,7 @@ export const TRUSTED_GATEWAY_URL = env.varOrDefault(
 export const ON_DEMAND_RETRIEVAL_ORDER = env
   .varOrDefault(
     'ON_DEMAND_RETRIEVAL_ORDER',
-    'ario-peer,trusted-gateway,chunks,tx-data',
+    's3,ario-peer,trusted-gateway,chunks,tx-data',
```

Comment:
It will only add `S3DataSource` if you instantiate the class (requires AWS variables to be set)

https://github.com/ar-io/ar-io-node/blob/5b33d7daa0333fd8a5bb8030d1c4981ffb1b08f8/src/system.ts#L304-L307